reference_guide/api_changes/api_changes_list_2019.md

```diff
@@ -29,6 +29,8 @@ See the note on how to document new problems on the main page reference_guide/ap
 `com.intellij.extapi.psi.MetadataPsiElementBase` class removed
 : Please use different base class for PSI elements.
 
+New method `UniqueVFilePathBuilder.hasFilesWithSameName` added. Used to enable editor tabs names disambiguation. 
```

Comment:
Firstly please follow the format described in [api_changes_list.md](https://raw.githubusercontent.com/JetBrains/intellij-sdk-docs/67d6da08f439ddc08a49f3a8f8dec71bda21b572/reference_guide/api_changes_list.md) file otherwise automatic processing will fail.
Secondly, why not just return `false` by default from the added method to avoid breaking API at all?